### ISSUE TITLE
[Task 5] Update tabs on the right

### ISSUE THREAD

**Issue by plhHKUST** [khwang0/2018F-COMP3111#83]
5.ii.a "Update all tabs on the right."
This means that we have to update also the results of "Summary" and "Table" when refine button is clicked?

Thank you

**Comment by comp3111ta** [comment]
Yes. But, if your group does not subscribe a particular task, for example, no one is doing Task 1 - Summary, then you can leave the Summary tab unchanged (very unlikely through)
